baseplate/experiments/targeting/tree_targeting.py

```diff
@@ -0,0 +1,197 @@
+from .base import Targeting
```

Comment:
nit: please include the boilerplate `from __future__` things on all new files so we can keep the py2/py3 differences minimal